x/mint/types/params.go

```diff
@@ -0,0 +1,94 @@
+package types
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store keys
+var (
+	KeyMintDenom     = []byte("MintDenom")
+	KeyBlocksPerYear = []byte("BlocksPerYear")
+)
+
+// ParamTable for minting module.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+func NewParams(
+	mintDenom string, inflationRateChange, inflationMax, inflationMin, goalBonded sdk.Dec, blocksPerYear uint64,
+) Params {
+	return Params{
+		MintDenom:     mintDenom,
+		BlocksPerYear: blocksPerYear,
+	}
+}
+
+// default minting module parameters
+func DefaultParams() Params {
+	return Params{
+		MintDenom:     sdk.DefaultBondDenom,
+		BlocksPerYear: uint64(60 * 60 * 8766 / 5), // assuming 5 second block times
```

Comment:
This should be an exported and documented constant btw (so the go-docs also document the assumptions around block times). I know this is still in draft status but I think we are going to have slower block times than 5s.

Reply:
Done